ci/test/00_setup_env_arm.sh

```diff
@@ -26,3 +26,4 @@ export GOAL="install"
 # -Wno-psabi is to disable ABI warnings: "note: parameter passing for argument of type ... changed in GCC 7.1"
 # This could be removed once the ABI change warning does not show up by default
 export BITCOIN_CONFIG="--enable-reduce-exports CXXFLAGS=-Wno-psabi --with-boost-process"
+export NO_WERROR=1
```

Comment:
which exactly errors? Is it a lot of them? because if it is disabled on CI, it means they can be increased over time

Reply:
There were enough fixes needed to open separate PRs for Clang and GCC reported warnings, they are being re-enabled after the fixes are implemented so they shouldn't accumulate further.